be/src/common/config.cpp

```diff
@@ -1407,6 +1407,9 @@ DEFINE_mBool(enable_sleep_between_delete_cumu_compaction, "false");
 
 DEFINE_mInt32(compaction_num_per_round, "1");
 
+DEFINE_mInt32(check_tablet_delete_bitmap_interval_seconds, "30");
```

Comment:
enable : interval > 0 
disable : interval <= 0

Reply:
180s or 360s is enough?

Reply:
It is better to use a separate configuration to control this feature